### PR TITLE
Update catalan localization messages.po

### PR DESCRIPTION
**Pull request by ivanbea**
New lines keep appearing, so new localization is done

You can check it @jordimas @darccio @ferranrego @MiquelAdell @mvdan @johnnydement @Mejans @surfdude29

### CODE REVIEW

File: src/locale/locales/ca/messages.po [hunk 1]
```diff
@@ -468,7 +468,7 @@ msgstr "Bloqueja el compte"
 
 #: src/view/com/profile/ProfileMenu.tsx:344
 msgid "Block Account?"
-msgstr ""
+msgstr "Bloqueja el compte"
```
Comment (jordimas) [inline src/locale/locales/ca/messages.po:472]: És una pregunta o afegim ? al final o una fórmula estil "Vols bloquejar el compte?"

File: src/locale/locales/ca/messages.po [hunk 2]
```diff
@@ -602,15 +602,15 @@ msgstr "per {0}"
...
+msgstr "Per {0}"
 
 #: src/view/com/profile/ProfileSubpageHeader.tsx:161
 msgid "by <0/>"
 msgstr "per <0/>"
 
 #: src/view/com/auth/create/Policies.tsx:87
 msgid "By creating an account you agree to the {els}."
-msgstr ""
+msgstr "Creant el compte indiques qu estàs d'acord amb {els},"
```
Comment (jordimas) [inline src/locale/locales/ca/messages.po:614]: que= que
al final cal posar "." no ","

Comment (ivanbea) [reply src/locale/locales/ca/messages.po:614]: fet

File: src/locale/locales/ca/messages.po [hunk 3]
```diff
@@ -922,7 +922,7 @@ msgstr "Configura els filtres de continguts per la categoria: {0}"
 
 #: src/components/moderation/ModerationLabelPref.tsx:116
 msgid "Configured in <0>moderation settings</0>."
-msgstr ""
+msgstr "Configurat a <0>configuració de moderació</0>"
```
Comment (surfdude29) [inline src/locale/locales/ca/messages.po:925]: I don't know if it matters here, but the full stop/period at the end is missing.

File: src/locale/locales/ca/messages.po [hunk 4]
```diff
@@ -1312,7 +1312,7 @@ msgstr "Descarta"
 
 #: src/view/com/composer/Composer.tsx:507
 msgid "Discard draft?"
-msgstr ""
+msgstr "DVols descartar l'esborrany?"
```
Comment (surfdude29) [inline src/locale/locales/ca/messages.po:1315]: It looks like there's a stray `D` at the beginning here?

File: src/locale/locales/ca/messages.po [hunk 5]
```diff
@@ -2024,11 +2024,11 @@ msgstr "Ves al pas anterior"
 
 #: src/view/screens/NotFound.tsx:55
 msgid "Go home"
-msgstr ""
+msgstr "Ves a l'inici"
```
Comment (surfdude29) [inline src/locale/locales/ca/messages.po:2027]: Comparing with the following string, it looks like there should be an accent on `Vés`?

File: src/locale/locales/ca/messages.po [hunk 6]
```diff
@@ -2045,27 +2045,27 @@ msgstr "Ves al següent"
...
+msgstr "Mitjans gràfics"
 
 #: src/view/com/modals/ChangeHandle.tsx:265
 msgid "Handle"
 msgstr "Identificador"
 
 #: src/lib/moderation/useReportOptions.ts:32
 msgid "Harassment, trolling, or intolerance"
-msgstr ""
+msgstr "Assetjament, troleig o intolerància""
```
Comment (surfdude29) [inline src/locale/locales/ca/messages.po:2056]: There's an extra quotation mark at the end here

File: src/locale/locales/ca/messages.po [hunk 7]
```diff
@@ -2451,7 +2451,7 @@ msgstr "Més informació sobre què és públic a Bluesky."
 
 #: src/components/moderation/ContentHider.tsx:152
 msgid "Learn more."
-msgstr ""
+msgstr "Mñes informació."
```
Comment (surfdude29) [inline src/locale/locales/ca/messages.po:2454]: Comparing to previous strings, it looks like there's a typo and the first word should be `Més`?

File: src/locale/locales/ca/messages.po [hunk 8]
```diff
@@ -3322,23 +3322,23 @@ msgstr "Obre la pantalla amb tots els canals desats"
 
 #: src/view/screens/Settings/index.tsx:696
 msgid "Opens the app password settings"
-msgstr ""
+msgstr "Obre la configuració de les contrasenyes d'apliació"
```
Comment (surfdude29) [inline src/locale/locales/ca/messages.po:3325]: Comparing to the superseded string below, I think there might be a typo in the last word with a missing `c` in `aplicació`

File: src/locale/locales/ca/messages.po [hunk 9]
```diff
@@ -4711,15 +4711,15 @@ msgstr "Ordena les respostes a la mateixa publicació per:"
...
 
 #: src/lib/moderation/useReportOptions.ts:65
 msgid "Spam"
-msgstr ""
+msgstr "Brossa"
 
 #: src/lib/moderation/useReportOptions.ts:53
 msgid "Spam; excessive mentions or replies"
-msgstr ""
+msgstr "Brosa; excessives mencions o respostes"
```
Comment (surfdude29) [inline src/locale/locales/ca/messages.po:4722]: Comparing to the string above (and after checking on Google Translate), I think there's a missing `s` from `Brossa`

File: src/locale/locales/ca/messages.po [hunk 10]
```diff
@@ -5076,7 +5076,7 @@ msgstr "Aquest contingut no es pot veure sense un compte de Bluesky."
 
 #: src/view/screens/Settings/ExportCarDialog.tsx:75
 msgid "This feature is in beta. You can read more about repository exports in <0>this blogpost</0>."
-msgstr ""
+msgstr "Aquesta funció està en versió beta. Podeu obtenir més informació sobre les exportacions de repositoris en <0>aquesta entrada de bloc</0>"
```
Comment (surfdude29) [inline src/locale/locales/ca/messages.po:5079]: There's a missing full stop/period at the end of the second sentence here

**Review by surfdude29 — COMMENTED** [review]
I just noticed a few small things, but well done for updating all those strings, that's a lot of work! 👌

File: src/locale/locales/ca/messages.po (hunk 5 above)
Comment (ivanbea) [reply src/locale/locales/ca/messages.po:2027]: thanks, "ves" is always without accent, I changed evereywhere now

**Comment by jordimas** [comment]
Hello

If you can please consider fixing the following mistakes in this PR:

msgid "Opens additional details for a debug entry"
msgstr "Obre detalls **adicionals** per una entrada de depuració"

adicionals -> addicionals


#: src/view/com/modals/DeleteAccount.tsx:203
msgid "Input password for account deletion"
msgstr "Introdueix la contrasenya per **elimiar** el compte"

elimiar -> eliminar

msgid "User list by you"
msgstr "Llista d'**usaris** feta per tu"

**Comment by ivanbea** [comment]
Moltes gràcies @jordimas, he aplicat les correccions que has indicat i les trobades a https://static.softcatala.org/quality/social-app.html (molt bona eina, entenc que es passa setmanalment i el 30 de març ho tornarà a revisar)

**Comment by jordimas** [comment]
Sí, cada setmana es recalcula, però clar, ho fa sobre el fitxer a master, llavors fins que no es faci merge d'aquesta PR no veurà res de nou (tampoc les correccions).

**Comment by ivanbea** [comment]
Entès, serà molt útil. Gràcies.

**Review by pfrazee — APPROVED** [review]
Thanks everyone!!

**Pull request merged**